lib/dogapi/facade.rb

```diff
@@ -59,6 +59,7 @@ class Client # rubocop:disable Metrics/ClassLength
     attr_accessor :v2
     # Support for API version 2.
 
+    # rubocop:disable Metrics/MethodLength, Metrics/LineLength
```

Comment:
Overall looks good to me other than disabling some rubocop options. Is there a way to make this method work with these turned on? It looks like you may need to break out your initialization into some smaller sections that handle different bits of the setup. Like a cloud ints intialize that does aws, gcp and azure for example.

Reply:
Thanks Matt! I worked with Nick on this in working out this solution. At the time, we couldn't find a real good reason to limit the method to only 32 lines and also were not sure on a clean way to make those “line too long” errors in there shorter without adding a bunch of lines to method.